.github/release-drafter.yml

```diff
@@ -0,0 +1,22 @@
+categories:
+  - title: '🚀 Features'
+    labels:
+      - 'feature'
```

Comment:
should we add `feat` also ? I see it in the documentation.

Reply:
Hm, we only have an "enhancement" label, also tracker seems to use that, too. I copied most of that config from the docs, so we might be able to drop 'feature', too. But I would wait and see if maybe that's there for a reason....

Reply:
:+1: